src/datalayer/reducers/auth.reducer.js

```diff
@@ -0,0 +1,27 @@
+import { authAction } from 'constants/actions';
+
+export const INITIAL_STATE = {
+  isLoginComponent: true,
+};
+
+export default (state = INITIAL_STATE, action) => {
+  console.log(state.isLoginComponent);
+  switch (action.type) {
+    case authAction.IS_LOGIN_COMPONENT: {
+      return {
+        ...state,
+        isLoginComponent: true,
```

Comment:
Not recommend (can not be extended). Here're some other solutions:
- `component`: "login" | "signup" | ...
- ...